web-common/src/features/dashboards/time-dimension-details/TDDTable.svelte

```diff
@@ -269,12 +269,16 @@
       setTimeout(pivot.draw, 0);
     }
   }
+  $: cssVarStyles = `--cursor: ${
+    comparing === "dimension" ? "pointer" : "default"
+  }`;
 </script>
 
 <div
   on:mouseleave={resetHighlight}
   style:height="calc(100% - 50px)"
-  class="w-full relative {comparing === 'dimension' && 'cursor-pointer'}"
+  style={cssVarStyles}
```

Comment:
Any specific reason to use direct style instead of class? If so a comment would be good here.